app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb

```diff
@@ -48,9 +48,14 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager < ManageI
 
   supports :catalog
   supports :create
+  supports :native_console
   supports :provisioning
   supports_not :volume_availability_zones
 
+  def console_url
+    "https://cloud.ibm.com"
```

Comment:
I noticed the others use `"https://#{endpoint.hostname}"`

Is there a reason why this needs to be hardcoded?